src/TemplateBuilder.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Staffim\SpreadsheetBuilder;
+
+use PhpOffice\PhpSpreadsheet\Reader;
+use PhpOffice\PhpSpreadsheet\RichText\RichText;
+use PhpOffice\PhpSpreadsheet\Writer;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+
+class TemplateBuilder
+{
+    protected const START_ROW = 0;
+    protected const START_COLUMN = 'A';
+
+    /**
+     * @var Spreadsheet
+     */
+    protected $template;
+
+    /**
+     * @var array
+     */
+    protected $rowErrors = [];
```

Comment:
there is no getter for that field. How it supposed to be use? Also, if we going to use this, we should add this in Table builder too. Lets add tests for exceptions

Reply:
removed